source/fundamentals/bson.txt

```diff
@@ -210,6 +210,29 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on any level, from
+``collection`` to ``Client``.
+
+The following example demonstrates how the ``BSONOptions`` is set using
+the ``Movies`` collection as an example.
```

Comment:
I: Two points from the Style Guide here: [Avoid passive voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/) and [Don't use "using" alone](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u:~:text=word%2C%20username.-,using,-by%20using%20with)

S:

```suggestion
The following example demonstrates how to set ``BSONOptions`` for 
the ``Movies`` collection:
```